apps/src/templates/sectionProgress/ScriptSelector.js

```diff
@@ -50,7 +50,6 @@ export default class ScriptSelector extends Component {
           onChange={event => onChange(parseInt(event.target.value))}
           style={styles.dropdown}
         >
-          <option key="default" value={''}/>
```

Comment:
removed this because clicking on the empty string option would cause all kinds of problems because it didn't set a scriptId, and the default script is now "Express Course"

Reply:
Thanks for doing this @Erin007 ! FYI @poorvasingal that there should no longer be an empty option in the progress tab's script selector.